network-api/networkapi/templates/pages/campaign-opportunity-base.html

```diff
@@ -0,0 +1,55 @@
+{% extends "./base-compiled.html" %}
+
+{% block bodyID %}{{ suffix }}{% endblock %}
+
+{% block content %}
+  <div class="container">
+    <div class="row">
+      <div class="py-5 {% if page.parent.title != parentTitle %}col-12 col-md-8 offset-md-3{% else %}col-12 col-lg-8 offset-lg-2{% endif %}">
+        {% if page.parent.title != parentTitle %}
+        <p class="h5-black">{{ page.parent.title }}</p>
+        {% endif %}
+        <h1 class="h1-white {% if page.parent.title != parentTitle %}hidden-sm-down{% endif %}">{% block h1 %}{% endblock %}</h1>
+      </div>
+    </div>
+  </div>
+  <div class="hidden-md-up mb-5" id="multipage-nav-mobile">
+    <div class="container">
+      <div class="row">
+        <div class="col-12"></div>
+      </div>
+    </div>
+  </div>
+  <div class="container">
+    <div class="row mb-5">
+      {% if page.parent.title != parentTitle %}
+      <div class="col-12 col-md-3 mb-3 hidden-sm-down">
+        <div class="multipage-nav" id="multipage-nav">
+          {% load pages_tags %}
+          {% page_menu "pages/menus/side-menu-items.html" %}
+        </div>
+      </div>
+      {% endif %}
+
+      {% block subcontent %}
+      <div class="{% if page.parent.title != parentTitle %}col-12 col-md-9{% else %}col-12 col-lg-8 offset-lg-2{% endif %}">
+        <div class="row">
+          <div class="cms col-12">{{ page.landingpage.content | safe }}</div>
+        </div>
+      </div>
+      {% endblock %}
+    </div>
+
+
+    <div class="row mb-5">
+      <div class="{% if page.parent.title != parentTitle %}col-md-12{% else %}col-12 col-lg-8 offset-lg-2{% endif %}">
```

Comment:
Oh, or does changing this to `col-lg-10 offset-lg-1` fix the width issue?

Reply:
Yeah, I think that does work better.